packages/node/src/protocol/install.ts

```diff
@@ -334,10 +334,10 @@ function computeStateChannelTransition(
   if (initiatorDepositTokenAddress !== responderDepositTokenAddress) {
     tokenIndexedBalanceDecrement = {
       [initiatorDepositTokenAddress]: {
-        [initiatorFbAddress]: initiatorBalanceDecrement
+        [initiatorFbAddress]: bigNumberify(initiatorBalanceDecrement)
```

Comment:
Do we need to recast again here and below if we've done it in the context of this call above already?